corehq/ex-submodules/casexml/apps/phone/data_providers/standard.py

```diff
@@ -46,6 +46,6 @@ def get_elements(self, restore_state):
             restore_state.user,
             restore_state.version,
             restore_state.last_sync_log,
-            restore_state.params.app
+            app=restore_state.params.app
```

Comment:
Thanks. I think my takeaway wasn't as much the bug as that I didn't pick it up. If I'd used Django's [test client](https://docs.djangoproject.com/en/1.9/topics/testing/tools/#the-test-client), it would have caught this. I'll do that in future.